app/src/main/res/values/strings.xml

```diff
@@ -143,18 +143,18 @@
     <string name="recovery_code_input_hint_10">Word 10</string>
     <string name="recovery_code_input_hint_11">Word 11</string>
     <string name="recovery_code_input_hint_12">Word 12</string>
-    <string name="recovery_code_error_empty_word">You forgot to enter this word.</string>
-    <string name="recovery_code_error_invalid_word">Wrong word.</string>
-    <string name="recovery_code_error_checksum_word">Your code is invalid. Please check all words as well as their position and try again!</string>
+    <string name="recovery_code_error_empty_word">You forgot to enter this word</string>
+    <string name="recovery_code_error_invalid_word">Wrong word</string>
+    <string name="recovery_code_error_checksum_word">Your code\'s invalid. Please check all words as well as their position and try again!</string>
     <string name="recovery_code_verification_ok_title">Recovery code verified</string>
-    <string name="recovery_code_verification_ok_message">Your code is correct and will work for restoring your backup.</string>
+    <string name="recovery_code_verification_ok_message">Your code\'s correct and will work when you need to restore.</string>
```

Comment:
Anywhere that we're just replacing is with apostrophe s feels off to me (personally).